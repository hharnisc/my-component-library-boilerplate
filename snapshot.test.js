import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  suit: 'Snapshots',
});
