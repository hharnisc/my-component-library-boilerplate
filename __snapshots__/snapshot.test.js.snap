// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots Text default 1`] = `
<span
  style={
    Object {
      "fontSize": "1rem",
    }
  }
>
  oh hey!
</span>
`;

exports[`Snapshots Text size=0.5 1`] = `
<span
  style={
    Object {
      "fontSize": "0.5rem",
    }
  }
>
  Rems = 0.5 = small
</span>
`;

exports[`Snapshots Text size=3 1`] = `
<span
  style={
    Object {
      "fontSize": "3rem",
    }
  }
>
  Rems = 3 = huge
</span>
`;
